mmdeploy/codebase/mmdet/deploy/object_detection.py

```diff
@@ -249,12 +262,16 @@ def get_preprocess(self, *args, **kwargs) -> Dict:
                 transforms[i]['size'] = transforms[i].pop('scale')
 
         data_preprocessor = model_cfg.model.data_preprocessor
+
         transforms.insert(-1, dict(type='DefaultFormatBundle'))
-        transforms.insert(
-            -2,
-            dict(
-                type='Pad',
-                size_divisor=data_preprocessor.get('pad_size_divisor', 1)))
+        if extra_pad is not None:
```

Comment:
Is it possible that both `RandomCenterCropPad` and `data_preprocessor` require pad?